subsys/bootloader/Kconfig

```diff
@@ -100,6 +100,7 @@ config SB_BPROT_IN_DEBUG
 
 config SB_CLEANUP_RAM
 	bool "Perform RAM cleanup"
+	default y
 	depends on !FW_INFO_PROVIDE_ENABLE
```

Comment:
Has the exact same issue as https://github.com/mcu-tools/mcuboot/pull/2451 has including destroying any data sharing capabilities because it doesn't even clear the correct RAM range

Reply:
EXT_API is not panned to be used on nRF54l devices.

Reply:
By data sharing, I mean data sharing in general, unrelated to ext APIs. E.g. a user can no longer have **any** memory retained between boots, nor can they share any information from application to bootloader or vice versa - and this is a feature customers and even FAEs are using

Reply:
I see, I will in fact remove the `default y`.
I just keep it for the time being to see which tests fail